core/src/main/java/com/linecorp/armeria/server/EmptyContentDecodedHttpRequest.java

```diff
@@ -102,33 +110,34 @@ public boolean isKeepAlive() {
 
     @Override
     public boolean isOpen() {
-        return delegate.isOpen();
+        return false;
     }
 
     @Override
     public boolean isEmpty() {
-        return delegate.isEmpty();
+        return true;
     }
 
     @Override
     public long demand() {
-        return delegate.demand();
+        return 0;
     }
 
     @Override
     public CompletableFuture<Void> whenComplete() {
-        return delegate.whenComplete();
-    }
-
-    @Override
-    public void subscribe(Subscriber<? super HttpObject> subscriber, EventExecutor executor) {
-        delegate.subscribe(subscriber, executor);
+        return completionFuture;
     }
 
     @Override
     public void subscribe(Subscriber<? super HttpObject> subscriber, EventExecutor executor,
                           SubscriptionOption... options) {
-        delegate.subscribe(subscriber, executor, options);
+        requireNonNull(subscriber, "subscriber");
+        requireNonNull(executor, "executor");
+        executor.execute(() -> {
```

Comment:
- Do we need to reschedule even when it is in the event loop?
- Don't we need to abort a late subscriber?

Reply:
> Don't we need to abort a late subscriber?

If so, we have to maintain the additional status.
I thought just allowing multiple subscribe would be much efficient instead of managing the status.
Also, we allowed multiple aggregate recently, so I thought multiple subscribe is not a big deal.